packages/Card/src/components/Separator/Separator.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Separator.styles";
+
+const propTypes = {
+  /** Size of the separator (min-height). */
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  size: ShirtSizes.SMALL,
+};
+
+const Separator = props => {
+  const { size } = props;
+
+  const separatorProps = {
+    size,
+  };
```

Comment:
Seems a bit weird here, what exactly are you trying to do here?

Reply:
Was attempting to keep things consistent with the way we set up the other components. However, it was kinda unnecessary so i removed it.